src/views/Dashboard/Customer/Home/GeneticData/Details/index.vue

```diff
@@ -83,20 +86,46 @@ export default {
     reasonOfRejection: false,
     trackingId: "",
     title: "",
-    message: ""
+    message: "",
+    orderDetail: null,
+    serviceDetail: null,
+    analysisDetail: null
   }),
 
   async mounted() {
     this.orderId = null
 
     if(this.$route.params.id) {
       this.orderId = this.$route.params.id
+      await this.getOrderDetail()
     }
 
     await this.getCancelFee()
   },
 
   methods: {
+    formatPrice(price) {
```

Comment:
maybe this function can be stored in global function

Reply:
done